controller/config.go

```diff
@@ -257,5 +265,41 @@ func LoadConfig(path string) (*Config, error) {
 		}
 	}
 
-	return config, nil
+	cfg.HealthChecks.BoltCheck.Interval = 30 * time.Second
+	cfg.HealthChecks.BoltCheck.Timeout = 20 * time.Second
+	cfg.HealthChecks.BoltCheck.InitialDelay = 30 * time.Second
+
+	errHolder := &errorz.ErrorHolderImpl{}
+	configMap := config.NewConfigMap(config.ToStringIntfMap(cfgmap), errHolder)
+
+	if healthCheckConfig := configMap.Child("healthChecks"); healthCheckConfig != nil {
+		if boltCheckConfig := configMap.Child("bolt"); boltCheckConfig != nil {
+			if val, found := boltCheckConfig.GetDuration("interval"); found {
+				cfg.HealthChecks.BoltCheck.Interval = val
+			}
+			if val, found := boltCheckConfig.GetDuration("timeout"); found {
+				cfg.HealthChecks.BoltCheck.Timeout = val
+			}
+			if val, found := boltCheckConfig.GetDuration("initialDelay"); found {
+				cfg.HealthChecks.BoltCheck.InitialDelay = val
+			}
+		}
+	}
+
+	if errHolder.HasError() {
+		return nil, err
+	}
+
```

Comment:
Here's the second style which is even more concise. Thoughts on if either, both or neither of these approaches look good?